src/rpc/tests.rs

```diff
@@ -259,6 +259,7 @@ async fn test_get_block() {
     assert_eq!(response, *Testnet2::genesis_block());
 }
 
+// This test Fails or Pass randomly because par_bridge does not guarantees the order of the iterator.
```

Comment:
```suggestion
// This test Fails or Pass randomly because par_bridge does not guarantee the order of the iterator.
```

Reply:
That comment is old and should not have been there at all. Thanks for pointing that out!